### PR TITLE
Cut v1.2.0

### PR DESCRIPTION
**Pull request by beorn7**
I believe the addition of the new query API is the only user-visible change.

All the internal changes about HTTP routing should not have any impact visible from the outside. Even the new metrics are only affecting the query API, IIRC.

@drumilpatel2000

### CODE REVIEW

**Comment by beorn7** [comment]
That was quick. I thought @drumilpatel2000 is already asleep. (o:

I'll change the release date to today then.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 1.2.0 / 2020-03-11
+
+* [FEATURE] Add an HTTP API to query metadata and runtime information. #184
```
Comment (roidelapluie) [inline CHANGELOG.md]: Maybe `HTTP API endpoint`?

Comment (roidelapluie) [reply CHANGELOG.md]: (If you change the date anyway..)

Comment (beorn7) [reply CHANGELOG.md]: It's actually two endpoints. I think talking about an API in general makes more sense.

Comment (beorn7) [reply CHANGELOG.md]: Actually, it should contain "metric" somehow, let me change that.

**Pull request merged**